galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -561,4 +565,21 @@ private void commandLineError(String message) {
                         + "Example Resource Management arguments: --obr infra.obr --obr test.obr --resourcemanagement");
         System.exit(-1);
     }
+
+    public boolean validateJavaLevel(){
+        String version = System.getProperty("java.version");
+        logger.trace("Checking version of Java, found: " + version);
+        if(version == null || version.isEmpty()){
+            logger.error("Unable to determine Java version - will exit");
+            return false;
+        }
+
+        if(version.startsWith("11")){
```

Comment:
Version 110.0 of java will be valid too. ;-)

Reply:
will fix that in time ;)